cmd/zt_generic_traverser_test.go

```diff
@@ -38,6 +40,43 @@ type genericTraverserSuite struct{}
 
 var _ = chk.Suite(&genericTraverserSuite{})
 
+// Test follow symlink functionality
+func (s *genericTraverserSuite) TestWalkWithSymlinks(c *chk.C) {
+	if runtime.GOOS == "windows" {
+		c.Skip("Windows does not allow true symlinks")
```

Comment:
This is not correct. It was correct many years ago, but now Windows has fine and real symlinks, see `mklink` command: https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/

So I would expect `os.Symlink` to work on Windows.  Is that not the case?

Reply:
Let's find out.

Reply:
Huh. It does work. Python on the other hand can't handle this behavior, so it won't persist into our smoke tests.